draft-ietf-jsonpath-base.md

```diff
@@ -296,10 +296,22 @@ stable JSON query language.
 ## Overview of JSONPath Expressions {#overview}
 
 JSONPath expressions are applied to a JSON value, the *argument*.
-Within the JSONPath expression, the abstract name `$` is used to refer
-to the *root node* of the argument, i.e., to the argument as a whole.
 
-JSONPath expressions use the *bracket notation*, for example:
+A JSONPath expression consists of an identifier followed by a series
+of zero or more segments each of which contains one or more selectors.
+
+### Identifiers
+
+The _root node identifier_ `$` refers to the *root node* of the argument,
+i.e., to the argument as a whole.
```

Comment:
no comma after i.e. I think?

Reply:
CMOS (the basis for RFC 7322) was in the "always comma after i.e. and e.g." camp last time I looked.
I have only the kindle version of CMOS 17, and that is so broken that I don't find the place where it says so.
I do find the occasional RFC that gets this wrong, overwhelmingly below RFC 5100.

Reply:
Found it:

```
i.e.; e.g. The first is the abbreviation for id est ("that is") ; the second is the abbreviation for exempli gratia ("for example") . The English equivalents are preferable in formal prose, though sometimes the compactness of these twocharacter abbreviations makes them de sirable. Always put a comma after ei
ther one. See also 6.20, 6.51.
```

(totally broken formatting, but very clear rule).

Reply:
bah snarl grumble. Having said that I'd rather be standardized than idiomatic.

I believe that written English should be consistent with the spoken form, and when I speak, I say “refers to the root node of the argument *pause* that is to the  argument as a whole" without a pause after "that is". But OK.

Reply:
Amazingly, I pause after the "that is" as well.

Of course, there is the general discussion of Fowler's "punctuation pitch" and the difference in US usage (pretty unanimously requiring the comma) and British usage (more varied).

But standardization is good, and the RFC editor has opined for CMOS, so there we are.